src/System.Device.Gpio/System/Device/Gpio/Drivers/RaspberryPi3LinuxDriver.cs

```diff
@@ -10,6 +10,8 @@
 using System.Threading;
 using System.Threading.Tasks;
 
+#pragma warning disable SA1011

```

Comment:
```suggestion
#pragma warning disable SA1011 // Closing square brackets should be spaced correctly https://github.com/DotNetAnalyzers/StyleCopAnalyzers/issues/2927
```
I would just add a comment directly in the code so that we remember to remove it when we don't need it any longer.